CHANGELOG.md

```diff
@@ -51,6 +51,17 @@ from `algosdk` and sends it to the network.
 - Fix `TxParams.noteb64` encoding - should use base64 decoder rather than TextEncoder.
 - Fix `ed25519verify` opcode implementation. Previously the signature was only checked against the data not the concatenation of "ProgData"||program||data. Additionally test scenarios was added to check the correct implementation. 
 
+### Breaking Changes
+
+#### @algo-builder/runtime
+
+- `executeTx` now returns `TxnReceipt[]` instead of `TxReceipt[]`.
+- `fundLsig` now returns `TxnReceipt[]` instead of `TxReceipt[]`.
```

Comment:
something wrong here

Reply:
Maybe lets add more info what's the difference etc.

Reply:
how do you suggest we do that, the only difference is instead of `AssetIndex` -> `asset-index`

Reply:
+ What's the difference b/w `TxnReceipt` & `TxReceipt` types? 
+ Also, please link (hyperlink) their respective type definitions.

Reply:
as I mentioned above the only difference is in key `AssetIndex` -> `asset-index`